### DISCUSSION TITLE
GitX User macOS Version Poll

### DISCUSSION THREAD

**Discussion by insha** [gitx/gitx#427] [Polls]
Hello GitX users! We're curious about the macOS versions our community is using. Please take a moment to participate in this quick poll. Your input helps us understand the preferences and needs of the GitX user community.

Thank you for participating!

**Comment by hannesa2** [comment]
My experience with https://github.com/gitx/gitx/discussions/419 tells me, that a tracking like Google analytics would be more efficient.
But I hope I'm wrong

**Reply by insha** [reply]
Definitely! I would like to avoid adding Google Analytics. Let me looking into a simple analytics if it is available; otherwise I can bounce some ideas off here.

Good thing is that we don’t need anything extensive. So something simple would do.
